specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json

```diff
@@ -0,0 +1,2059 @@
+        },
+        "ConnectionGatewayPropertiesDefinition": {
+            "description": "The gateway properties",
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "connectionGatewayInstallation": {
+                    "$ref": "#/definitions/ConnectionGatewayInstallationReference"
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "#/definitions/GatewayPropertiesDefinition"
+                }
+            ]
+        },
+        "ConnectionGatewayInstallationReference": {
+            "description": "The gateway installation reference",
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "location": {
+                    "description": "Resource reference location",
+                    "type": "string"
+                },
+                "name": {
+                    "description": "Resource reference name",
+                    "type": "string"
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "#/definitions/ResourceReference"
+                }
+            ]
+        },
+        "ResourceReference": {
+            "type": "object",
+            "x-abstract": true,
+            "additionalProperties": false,
+            "properties": {
+                "id": {
+                    "description": "Resource reference id",
+                    "type": "string"
+                },
+                "type": {
+                    "description": "Resource reference type",
+                    "type": "string"
+                }
+            }
+        },
+        "GatewayPropertiesDefinition": {
+            "type": "object",
+            "x-abstract": true,
+            "additionalProperties": false,
+            "properties": {
+                "contactInformation": {
+                    "description": "The gateway admin",
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "displayName": {
+                    "description": "The gateway display name",
+                    "type": "string"
+                },
+                "description": {
+                    "description": "The gateway description",
+                    "type": "string"
+                },
+                "machineName": {
+                    "description": "The machine name of the gateway",
+                    "type": "string"
+                },
+                "status": {
+                    "description": "The gateway status",
+                    "type": "object"
+                },
+                "backendUri": {
+                    "description": "The URI of the backend",
+                    "type": "string"
+                }
+            }
+        },
+        "ResourceDefinition": {
```

Comment:
is there an existing common model we can use? not sure what the practice in this repo is.

Reply:
This is the only file for this API version, so I can't snag it from anywhere else.

Reply:
yeah, let's follow what other existing files are doing.